packages/engine/bin/cli/src/main.rs

```diff
@@ -66,6 +62,21 @@ pub enum ExperimentType {
     // Generate shell completitions
 }
 
+impl From<ExperimentType> for orchestrator::ExperimentType {
+    fn from(t: ExperimentType) -> Self {
+        match t {
+            ExperimentType::SimpleExperiment(simple) => orchestrator::ExperimentType::Simple {
```

Comment:
Is this necessary because of clap?

Reply:
Yes, I didn't want to pollute the `orchestrator` with `clap` annotations. `cli::ExperimentType` implements the types needed for using it in the CLI as arguments, `orchestrator::ExperimentType` does not implement those traits.

Reply:
Makes sense, we might want to revisit it at some point